firmware_DavidBoard/Core/Src/TemperatureCalc.c

```diff
@@ -0,0 +1,94 @@
+/*
+ * TempCalc.c
+ *
+ *  Created on: Mar 13, 2022
+ *      Author: JacobS
+ */
+
+#include "TemperatureCalc.h"
+
+/***********************************************************************************************************************
+ * Definitions
+ **********************************************************************************************************************/
+
+/***********************************************************************************************************************
+ * Variables
+ **********************************************************************************************************************/
+
+// HAL handle
+/*extern I2C_HandleTypeDef hi2c2;
+static uint8_t rawAdcData[16];
+static bool adcDataNew;
+static I2CBusStatus_t currentBusStatus;*/
+
+
+//static double V1;
+//static double R1;
+//static double T_K_inverse;
+static double TCelsius;
+static int V1000int;
```

Comment:
why declare these variables here ? They are only used by a single function. The point of having module variables like this is to allow multiple functions inside a single module to communicate. naming convention should be camelCase with the first letter lower case. I thought these were macros.